docs/FAQ.md

```diff
@@ -259,12 +259,16 @@ efficient to stay on the CPU instead of going back and forth.
 
 Yes, DPP still works.  It might not be as efficient as it could be, and we are working to improve it.
 
+DPP is not supported on Databricks with the plugin.
```

Comment:
There needs to be more detail here.  Will a query crash if DPP is enabled or will it run to completion and correctly but with less performance?

Reply:
@jlowe I added the line saying DPP will not fail query but queries can not benefit.